atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServices.java

```diff
@@ -59,7 +59,19 @@ private CassandraKeyValueServices() {
         // Utility class
     }
 
-    static void waitForSchemaVersions(Cassandra.Client client, String tableName, int schemaTimeoutMillis)
+    static void waitForSchemaVersions(
+            Cassandra.Client client,
+            String tableName,
+            int schemaTimeoutMillis)
+            throws TException {
+        waitForSchemaVersions(client, tableName, schemaTimeoutMillis, false);
```

Comment:
createTable aside, which I believe is a fixable issue in the world you're describing, I believe truncate() at least is not fixable and becomes incorrect if not done at ALL and a node just tries to catch up on schema mutations later.